ports/espressif/bindings/espnow/__init__.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2023 MicroDev
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "shared-bindings/util.h"
+
+#include "bindings/espnow/__init__.h"
+#include "bindings/espnow/ESPNow.h"
+#include "bindings/espnow/ESPNowPacket.h"
+#include "bindings/espnow/Peer.h"
+#include "bindings/espnow/Peers.h"
+
+//| """ESP-NOW Module
+//|
+//| The `espnow` module provides an interface to the
+//| `ESP-NOW <https://www.espressif.com/en/products/software/esp-now/overview>`_
+//| protocol provided by Espressif on its SoCs
+//| (`API docs <https://docs.espressif.com/projects/esp-idf/en/latest/api-reference/network/esp_now.html>`_).
+//|
+//| **Sender**
+//|
+//| .. code-block:: python
+//|
+//|     import espnow
+//|
+//|     e = espnow.ESPNow()
+//|     peer = espnow.Peer(mac=b'\xaa\xaa\xaa\xaa\xaa\xaa')
+//|     e.peers.append(peer)
```

Comment:
Should a peer always know what network it is a part of? You could do this then and have it auto-add to the peers list.
```suggestion
//|     e = espnow.ESPNow()
//|     peer = espnow.Peer(e, mac=b'\xaa\xaa\xaa\xaa\xaa\xaa')
```

Reply:
Not sure what exactly you meant here. Only the sender needs to know the `peer`.